addons/test_website/tests/test_redirect.py

```diff
@@ -231,3 +232,91 @@ def test_05_redirect_404_notfound_record(self):
         r = self.url_open(url_rec1, allow_redirects=True)
         self.assertEqual(r.status_code, 200)
         self.assertTrue(r.url.endswith(redirect.url_to))
+
+
+@tagged('-at_install', 'post_install')
+class TestRedirectPortal(HttpCase):
```

Comment:
If we end up removing "test_customer_access_not_logged_language_redirection", would rename to "TestPortalRedirectionPerformance", as this one seems more performance-oriented.